### ISSUE TITLE
Error on export return code: 43

### ISSUE THREAD

**Issue by josecupchik** [olivierkes/manuskript#886]
I get this message during preview for PDF but this export the file, but no all the time. 
![Screenshot from 2021-06-08 12-06-17](https://user-images.githubusercontent.com/84065811/121157209-057ec680-c852-11eb-8d9a-02dc779c73d5.png)

**Comment by TheJackiMonster** [comment]
Yes, I also experienced this. The problem about fixing this is that we had a quite similar problem because this happens basically everytime Pandoc changes its parameters or specs. So I would actually like to see a fix for this which prevents it to happen again in the future but I'm not sure if that is possible.

**Comment by obw** [comment]
What about a INI file with the parameters for pandoc?

When we use sections, we can also support multiple Versions of pandoc.

Could be something like that:

```
[pandoc.ver 1.19.2.4]
toc-depth={{toc-depth}}
--standalone=__NONE__
...

[pandoc.ver 1.19.2.5]
...
```

If `__NONE__`, then no parameter.
If in two curly braces, then var replacement and a parameter with value.
Everything else parameter, which should use the given value.

When there is a new pandoc version, which makes trouble, only the ini must be corrected, in a step later, we could make something like an updater only for this file. Should be easy. So no coding is needed and when someone is forced to use an older version of pnadoc, it's also working, because the parameter collection, is avail in the ini.

Regards

P.S.: We could also use wildcards and defaults with this System, but it will make more work to implement.

**Comment by TheJackiMonster** [comment]
> What about a INI file with the parameters for pandoc?

Yes, that sounds like a good solution. So we good properly inform the user about a Pandoc version mismatch if a newer configuration on out side is missing. We should definitely not hardcore this as before every time Pandoc changes and becomes incompatible with Manuskript.

**Comment by josecupchik** [comment]
If I understand correctly, I have to install an older version of Pandoc. 
How can I do it?

**Comment by TheJackiMonster** [comment]
Technically there are all releases as downloads on Pandocs Github repository: https://github.com/jgm/pandoc/releases

But I'm not sure currently which is the last compatible version.

**Comment by josecupchik** [comment]
Sure, I hope some other more concrete solution is found

Thanks

Josef

**Comment by TheJackiMonster** [comment]
As mentioned in issue #922 I have now implemented a change to pick options for Pandoc depending on the version of the installed binary. So this issue should be solved with future releases without any user going to the Pandoc release page manually.
